CHANGELOG.md

```diff
@@ -1,6 +1,44 @@
-## 3.5.0 (Unreleased)
+## 3.5.0 (TBC)
 
-Changes in progress for v3.5.0 are available at [.changes/v3.5.0](https://github.com/vmware/terraform-provider-vcd/tree/master/.changes/v3.5.0) until the release.
+## FEATURES
+* **New Resource:** `vcd_nsxt_alb_settings` for managing NSX-T ALB on NSX-T Edge Gateways ([#726](https://github.com/vmware/terraform-provider-vcd/pull/726))
+* **New Data source:** `vcd_nsxt_alb_settings` for reading NSX-T ALB on NSX-T Edge Gateways  ([#726](https://github.com/vmware/terraform-provider-vcd/pull/726))
+* **New Resource:** `vcd_library_certificate` for managing certificates in library (provider configuration) ([#733](https://github.com/vmware/terraform-provider-vcd/pull/733))
+* **New Data source:** `vcd_library_certificate` for reading certificates in library ([#733](https://github.com/vmware/terraform-provider-vcd/pull/733))
+* **New Resource:** `vcd_nsxt_alb_edgegateway_service_engine_group` for managing NSX-T ALB Service Engine Groups
+  assignments to Edge Gateways ([#738](https://github.com/vmware/terraform-provider-vcd/pull/738), [#764](https://github.com/vmware/terraform-provider-vcd/pull/764))
+* **New Data source:** `vcd_nsxt_alb_edgegateway_service_engine_group` for reading NSX-T ALB Service Engine Groups
+  assignments to Edge Gateways ([#738](https://github.com/vmware/terraform-provider-vcd/pull/738), [#764](https://github.com/vmware/terraform-provider-vcd/pull/764))
+* **New Resource:** `vcd_vdc_group` for managing VDC groups ([#752](https://github.com/vmware/terraform-provider-vcd/pull/752))
+* **New Data source:** `vcd_vdc_group` for reading VDC groups ([#752](https://github.com/vmware/terraform-provider-vcd/pull/752))
+* **New Resource:** `vcd_nsxt_alb_pool` for NSX-T Load balancer pools ([#756](https://github.com/vmware/terraform-provider-vcd/pull/756))
+* **New Data source:** `vcd_nsxt_alb_pool` for reading NSX-T Load balancer pools ([#756](https://github.com/vmware/terraform-provider-vcd/pull/756))
+* **New Resource:** `vcd_nsxt_alb_virtual_service` for managing NSX-T ALB Virtual Service on NSX-T Edge Gateways
+  ([#757](https://github.com/vmware/terraform-provider-vcd/pull/757), [#764](https://github.com/vmware/terraform-provider-vcd/pull/764))
+* **New Data source:** `vcd_nsxt_alb_virtual_service` for reading NSX-T ALB Virtual Service on NSX-T Edge Gateways
+  ([#757](https://github.com/vmware/terraform-provider-vcd/pull/757), [#764](https://github.com/vmware/terraform-provider-vcd/pull/764))
+
+## IMPROVEMENTS
+* Remove Coverity warnings from code ([#734](https://github.com/vmware/terraform-provider-vcd/pull/734))
+* Add support for disk controller type nvme ([#680](https://github.com/vmware/terraform-provider-vcd/pull/680),[#739](https://github.com/vmware/terraform-provider-vcd/pull/739))
```

Comment:
I think we should have the corresponding resource name at the beginning of the sentence (like a few lines below).

Reply:
Added.